mkosi

```diff
@@ -1295,7 +1296,7 @@ def prepare_tree(args: CommandLineArguments, root: str, do_run_build_script: boo
         os.mkdir(os.path.join(root, "etc/kernel"), 0o755)
 
         with open(os.path.join(root, "etc/kernel/cmdline"), "w") as cmdline:
-            cmdline.write(' '.join(args.kernel_command_line))
+            cmdline.write(' '.join(args.kernel_command_line + ["console=ttyS0"] if args.qemu_headless else []))
```

Comment:
This line still needs to go, because it would clash with the current HEAD of master.

Reply:
Omg

Reply:
should be fixed